Google.Api.Gax.Grpc/ClientBuilderBase.cs

```diff
@@ -537,6 +545,7 @@ protected virtual void Configure(IServiceProvider provider)
         {
             GaxPreconditions.CheckNotNull(provider, nameof(provider));
 
+            LoggerFactory ??= provider.GetService<ILoggerFactory>();
```

Comment:
Hmm. Not sure - let's have a think about that separately. (We could make the Configure method public instead of protected. I can't immediately remember why it's protected anyway, as we'd need to generate code to call it... let's discuss.)